composer.json

```diff
@@ -12,7 +12,7 @@
     "require": {
         "drupal/commerce_shipping": "^8.2",
         "openlss/lib-array2xml": "^0.5.1",
-        "thejacer87/canadapost-api": "dev-master"
+        "thejacer87/canadapost-api": "^0.1.0"
```

Comment:
Should be `^0.1`. When we deliver the tracking API feature we will pump it to `0.2` as it's a feature not a patch and it won't get picked up.